pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/PdfBucketService.java

```diff
@@ -24,7 +24,7 @@ public class PdfBucketService {
     private final boolean useCloudStorage;
 
     public static String getBlobName(String umbrellaGuid, String studyGuid, String userGuid, String pdfConfigName, String pdfVersionTag) {
-        return String.format("%s/%s/%s_%s_%s.pdf", umbrellaGuid, studyGuid, userGuid, pdfConfigName, pdfVersionTag);
+        return String.format("%s/%s/%s_%s_%s.pdf", umbrellaGuid, studyGuid.toLowerCase(), userGuid, pdfConfigName, pdfVersionTag);
```

Comment:
Can we either add documentation to the method indicating the downcasing behavior, or change this so that the caller is responsible for performing the downcasing, and this method just uses the string as-is?